manifest.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: apps/v1
```

Comment:
Question: how did you test this specific manifest?  Which OpenShift cluster version(s) did you apply this manifest to, and did you check that the plugin is enabled and were our pages populated?  Take note that we should document this in the readme as part of the documentation story.